index.html

```diff
@@ -0,0 +1,81 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>abdulsomodtribute page</title>
+    <link rel="stylesheet" href="stylesheet.css" />
+  </head>
+
+  <body>
+    <main id="main">
+      <h1 id="main-head">Robot technology</h1>
+      <p id="title">Robotics reserch what is use for and more information</p>
+      <figure id="img-div">
+        <img id="image" src="robot.jpg" alt="Robotics information" />
+        <figcaption id="img-caption">
+          humanoid robot this is the first robot that was maid and it is White in color
+        </figcaption>
```

Comment:
Let's punctuate the text and also use appropriate spacing (`padding`/`margin`). 

Kindly do this for all other occurrences